File: app/src/main/java/com/scottmangiapane/cardclicks/OnAppStart.java
package com.scottmangiapane.cardclicks;

import android.app.Application;
import android.content.Context;
import android.content.SharedPreferences;

import androidx.preference.PreferenceManager;

public class OnAppStart extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Context context = getApplicationContext();
        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(context);
        SharedPreferences.Editor editor = sp.edit();
        editor.putInt("launch_count", sp.getInt("launch_count", 0) + 1);
        editor.apply();
    }
}

File: app/src/main/java/com/scottmangiapane/cardclicks/OnAppStart.kt
package com.scottmangiapane.cardclicks

import android.app.Application
import androidx.preference.PreferenceManager
import androidx.core.content.edit

class OnAppStart : Application() {
    override fun onCreate() {
        super.onCreate()
        val context = applicationContext
        val sp = PreferenceManager.getDefaultSharedPreferences(context)
        sp.edit {
            putInt("launch_count", sp.getInt("launch_count", 0) + 1)
        }
    }
}
